ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -117,15 +139,99 @@ type opennebula_tm_mad_conf = {
     "ln_target" : string = "NONE"
     "clone_target" : string = "SYSTEM"
     "shared" : boolean = true
+    "ds_migrate" ? boolean
+} = dict();
+
+@documentation{
+The  configuration for each driver is defined in DS_MAD_CONF.
+These values are used when creating a new datastore and should not be modified
+since they defined the datastore behavior.
+}
+type opennebula_ds_mad_conf = {
+    @{name of the transfer driver, listed in the -d option of the DS_MAD section}
+    "name" : string = "dummy"
+    @{comma separated list of required attributes in the DS template}
+    "required_attrs" : string[] = list('')
```

Comment:
I'm not quite sure.. I'm using the same oned.conf schema provided by opennebula-server v5 package. So they keep the option even if the list is empty `required_attrs = ""`, so I'm using the same conf schema. 
This is not documented: http://docs.opennebula.org/5.0/deployment/references/oned_conf.html but I can ask if it's optional or not, in any case is safe to use the same conf, just in case :smile: